includes/class-wc-form-handler.php

```diff
@@ -630,8 +631,26 @@ public static function order_again() {
 
 		do_action( 'woocommerce_ordered_again', $order->get_id() );
 
+		$num_items_in_cart = count( WC()->cart->get_cart() );
```

Comment:
The cart gets emptied on line 587 so my reasoning was that would be a non-issue. Am I correct in this assumption?